pandas/tests/reshape/test_util.py

```diff
@@ -4,31 +4,101 @@
 import pandas.util.testing as tm
 from pandas.core.reshape.util import cartesian_product
 
+from hypothesis import strategies as st
+from hypothesis import given, settings, assume
+from datetime import date
+from dateutil import relativedelta
+import string
+
+
+NO_OF_EXAMPLES_PER_TEST_CASE = 20
+
+
+def get_elements(elem_type):
+    strategy = st.nothing()
+    if elem_type == bool:
+        strategy = st.booleans()
+    elif elem_type == int:
+        strategy = st.integers()
+    elif elem_type == float:
+        strategy = st.floats()
+    elif elem_type == str:
+        strategy = st.text(string.ascii_letters, max_size=10)
+    return strategy
+
+
+@st.composite
+def get_seq(draw, types, mixed=False, min_size=None, max_size=None, transform_func=None):
+    """helper function to generate strategy for creating lists. parameters define the nature of to be generated list.
```

Comment:
These lines are too long (check the log output or run flake8 locally)

Reply:
Also, we use the numpy docstring standard: http://pandas-docs.github.io/pandas-docs-travis/contributing_docstring.html